tests/testthat/test-get.R

```diff
@@ -226,7 +226,7 @@ test_that("get_packages() works as expected with current return value", {
     )
   )
   expect_identical(
-    get_packages(client, page_size = 2, limit = 2),
+    get_packages(client),
```

Comment:
What was `limit = 2` here before? I don't see that being removed, but maybe it was just eaten in `...` somewhere?

Regardless, this is one place where having the ability to specify makes testing easier (in a way that encourages good factoring of our own code...). This particular parameter is just passed through so probably NBD to have a test that confirms that, but if there was any sort of processing or whatever it certainly would make something like that easier.

Reply:
[This](https://github.com/posit-dev/connectapi/pull/378/files#r1992153156) is where `limit` was removed from.

---

Side note, slightly related: `connectapi` used to paginate until the API returned empty pages, which required explicitly setting the limit at 2 in the tests. The pagination quirk has now been fixed: paginating until the response is empty is not required because the responses provide the `total` number of objects to be returned.